src/object/object_printer.cpp

```diff
@@ -1057,6 +1060,17 @@ void object_printer::describe_class (struct db_object *class_op)
       m_buf (", COLLATE %s", class_descr.collation);
     }
 
+  /* tde_algorithm */
+  if (sm_get_class_tde_algorithm (class_op, &tde_algo) == NO_ERROR)
+    {
+      if (tde_algo != TDE_ALGORITHM_NONE)
+	{
+	  tde_algo_str = tde_get_algorithm_name (tde_algo);
+	  assert (tde_algo_str != NULL);
+	  m_buf (" ENCRYPT %s", tde_algo_str);
+	}
+    }
+
```

Comment:
not needed processing for error?

Reply:
I followed surrounding codes like calling `sm_is_reuse_oid_class()` above. Is it needed?

Reply:
Ok, I see.